src/Controls/tests/TestCases.Shared.Tests/UITest.cs

```diff
@@ -18,15 +21,20 @@ namespace Microsoft.Maui.TestCases.Tests
 #elif WINTEST
 		[TestFixture(TestDevice.Windows)]
 #endif
-	public abstract class UITest : UITestBase
+	public abstract class UITest 
 	{
-		protected const int SetupMaxRetries = 1;
```

Comment:
This has never been enabled since we started using this testing harness, so I'm just going to delete it for now.